### PR TITLE
SDM530 - Added Resetable Active Memory

### PR DESCRIPTION
**Pull request by Jaaxy**
## Description:

I added the resetable Active Import/Export Memory, because the SDM530 doesn´t support the 3 Phases Import/Export like the SDM630.

The SDM530 does only accept the resetable ActivePower Import/Export in summary of all three phases.

**Related issue (if applicable):** NA

## Checklist:
  - [x] The pull request is done against the latest development branch
  - [x] Only relevant files were touched
  - [x] Only one feature/fix was added per PR and the code change compiles without warnings
  - [x] The code change is tested and works on Tasmota core ESP8266 V.2.7.4.9
  - [ ] The code change is tested and works with core ESP32 V.1.0.6
  - [x] I accept the [CLA](https://github.com/arendst/Tasmota/blob/development/CONTRIBUTING.md#contributor-license-agreement-cla).

_NOTE: The code change must pass CI tests. **Your PR cannot be merged unless tests pass**_

### CODE REVIEW

**Comment by ascillato** [comment]
Hi, your PR has conflicts with the actual codebase. May be you used an old version?
Please, can you solve those conflicts? Thanks.

**Review by Jason2866 — CHANGES_REQUESTED** [review]
Driver is enabled by default please do not change
https://github.com/arendst/Tasmota/pull/11818/files#diff-f2a16bbe8989c2d8613ad0a8008616219427637e2314796a6f4acd9e14efad22R723

**Comment by ascillato2** [comment]
@Jaaxy 

Hi, thanks for your work, but as it is can not be merged nor tested. If this PR remains in this condition for some more days, it will be automatically closed for being stale by the stale bot.

Are you able to check the conflicts with the actual codebase? Seems that you have used an old version of Tasmota for your changes.

**Comment by ascillato2** [comment]
@Jaaxy

I have fixed in your fork the conflicts and also disabled by default the SDM630 and SDM530 drivers. They should be enabled only when the user needs them.

You don't need to do anything else on this PR. All fixed.

**Comment by Jaaxy** [comment]
@ascillato2 
Thx for fixing.

File: tasmota/i18n.h [hunk 1]
```diff
@@ -70,7 +70,7 @@
 #define D_JSON_ERROR "Error"
 #define D_JSON_EVENT "Event"
 #define D_JSON_EVERY "Every"
-#define D_JSON_EXPORT "Export"
```
Comment (ascillato) [inline tasmota/i18n.h:73]: This Shouldn't be deleted.

**Review by ascillato — CHANGES_REQUESTED** [review]

File: tasmota/i18n.h (hunk 1 above)
Comment (Jaaxy) [reply tasmota/i18n.h:73]: Entry is double in List

look at Line -214 #define D_JSON_EXPORT "Export"

**Pull request closed without merging**